internal/agent/hooks/gruboptions.go

```diff
@@ -11,6 +11,9 @@ import (
 type GrubOptions struct{}
 
 func (b GrubOptions) Run(c config.Config, _ v1.Spec) error {
+	if len(c.Install.GrubOptions) == 0 {
```

Comment:
this just skips runnig this if we dont have any options, makes no sense to run on empty options